mw-webapp/src/component/editableText/EditableNumber.tsx

```diff
@@ -0,0 +1,76 @@
+import {useState} from "react";
+import {renderSpan} from "src/component/editableText/renderSpan";
+import {Input} from "src/component/input/Input";
+
+/**
+ * Cell item props
+ */
+interface EditableNumberProps {
+
+  /**
+   * Cell item's number
+   */
+  value: number;
+
+  /**
+   * Function that update element on Enter click or unfocused
+   */
+  onChangeFinish: (value: number) => void;
+}
+
+/**
+ * Render Input or span depend on client actions
+ */
+export const EditableNumber = (props: EditableNumberProps) => {
+  const [isEditing, setIsEditing] = useState(false);
+  const [value, setValue] = useState(props.value);
+
+  /**
+   * HandleChangeText
+   */
+  const handleChangeNumber = () => {
```

Comment:
Unconsistent naming and value, be careful. Better to call it handleChangeFinish, think why. The same for similar places

Reply:
Renamed